### PR TITLE
Add imagery to software delivery page

### PR DESCRIPTION
**Pull request by mattgraygithub**
This adds the images, CTA link and metarobots indexing for the Software Delivery page as per [Leankit ticket 221](https://codurance-online.leankit.com/card/1069431621)

### CODE REVIEW

File: src/_data/service_lines_tabber.yml [hunk 1]
```diff
@@ -84,7 +84,7 @@ en:
         - Achieve value-driven prioritization
         - Innovate through experimentation
       details: Build MVPs, Proof of Concepts, and Prototypes; Create a high-level technical strategy and roadmap; Customer journey mapping; Value stream mapping; Service blueprint and design; Product analytics; Usability testing; AB Testing; User research
-      image-path: /assets/custom/img/service-lines/service-line-example/tabber-placeholder-a.svg
+      image-path: /assets/custom/img/software-delivery/tabber-decorations_tabber-product-strategy.svg
```
Comment (LL782) [inline src/_data/service_lines_tabber.yml:87]: I'm a bit surprise you're not using the `service-lines` directory, e.g. `/assets/custom/img/service-lines/software-delivery/tabber-decorations_tabber-product-strategy.svg`

Not a biggie just trying to get on the same page for consistency

File: src/services/software-delivery/index.html [hunk 2]
```diff
@@ -6,17 +6,17 @@
 description-es: Te ayudamos a definir y desarrollar las características que necesitas proporcionando equipos multifuncionales o la capacidad adicional requerida para hacer entregas a escala.
 image:
    src: /assets/custom/img/service-lines/service-line-example/meta-social-image.png
```
Comment (LL782) [inline src/services/software-delivery/index.html]: Looks like we're still using the placeholder image here

**Review by LL782 — APPROVED** [review]
Nice work @mattgraygithub :+1:

**Comment by LL782** [comment]
✋ Just took a look at the preview link and I'm still seeing the placeholder SVGs

**Review by LL782 — CHANGES_REQUESTED** [review]
@mattgraygithub looks like the tabber SVGs for ES version still need to be updated

File: src/_data/service_lines_tabber.yml (hunk 1 above)
Comment (mattgraygithub) [reply src/_data/service_lines_tabber.yml:87]: The image folders are still a bit untidy with softmod images in 4 different places. I'm going to do some clean up on a separate branch

**Review by LL782 — APPROVED** [review]
Latest changes look good in code :+1: 
Please note I haven't checked in the browser myself

**Pull request merged**